### DISCUSSION TITLE
Technical bug prevent me from re-verify my student status

### DISCUSSION THREAD

**Discussion by phmngcthanh** [community/community#120118] [GitHub Education]
### Hi everyone,

Hi,
I'm re-applying for Github student pack. Unfortunately, some technical problem prevent me  to get through second stage of verifying. 
When i have selected my user type, and system said "You have verified the email address ...." so i dont have to type my email again ( and dont have any field to type)
![image](https://github.com/community/community/assets/18462809/1503275d-37cd-4fa5-9d56-daf9fb3e179d)

But when i clicked "continue" it said error "User type can't be blank We require applicants to use a school-issued email address to apply."
![image](https://github.com/community/community/assets/18462809/111e869e-adc4-4f9c-aa2f-e2a024bb3b7f)

Any help is appreciated

**Comment by Unnati2603** [comment]
same problem

**Comment by himal28924** [comment]
I also have the same problem , even tried from incognito tab still does not work. 
![image](https://github.com/community/community/assets/41054866/0f5b7393-fcf7-4131-9e76-49885de3ebdc)

**Comment by phmngcthanh** [comment]
What i've tried:
1, use incognito - didnt work
2, disable adblock - didnt work

**Reply by himal28924** [reply]
Do you know if we can apply for benefit from GitHub app? I could not find it .

**Reply by phmngcthanh** [reply]
i didn’t found the option to verify from Github iOS. But something weird in safari:
Safari’s default permission handling for website location is Ask:
![IMG_2232](https://github.com/community/community/assets/18462809/75f916c4-2bce-419b-9727-4ed28a413f7d)

 but it didn’t bother to ask me, it just said:
![IMG_2230](https://github.com/community/community/assets/18462809/2b13d7e5-fad2-44cb-935c-5bb406b0e251)


Allow site’s permission in safari: didnt work 
Turn off Icloud Relay: didnt work

**Comment by BatuToy** [comment]
İ also have the same problem . It still doesn't work !

**Comment by david-s-martinez** [comment]
same here!

**Reply by david-s-martinez** [reply]
Now it worked for me. However, in the next page when I upload the image for my university certificate and I press "Process my application" it just refreshes the page. Is anyone facing the same issue?
